test/proxy.deltas.test.ts

```diff
@@ -0,0 +1,372 @@
+import {ProxyDeltasTestFactory} from "../contract-bindings/ethers";
+import {ProxyDeltasTest} from "../contract-bindings/ethers/ProxyDeltasTest";
+import buidler from "@nomiclabs/buidler";
+import {BigNumber, BigNumberish} from "ethers";
+import {assert} from "chai";
+import {submitFailing} from "./support";
+
+async function deployProxyDeltasTest(): Promise<ProxyDeltasTest> {
+  const [signer] = await buidler.ethers.getSigners();
+  return new ProxyDeltasTestFactory(signer)
+    .deploy()
+    .then((deltas) => deltas.deployed());
```

Comment:
I’d prefer to use `await` consistently.

```suggestion
  const deltas = await new ProxyDeltasTestFactory(signer).deploy();
  await deltas.deployed();
```

Reply:
Me too, fixed